bin/upsert_library_model.py

```diff
@@ -0,0 +1,36 @@
+import json
+import sys
+
+import requests
+import yaml
+from tenacity import retry, stop_after_attempt, wait_fixed
+
+
+@retry(wait=wait_fixed(20), stop=stop_after_attempt(3), reraise=True)
+def upsert_pretrained_model(api_key, model_slug, commit_sha, baseten_url):
+    url = f"{baseten_url}/upsert_pretrained_model"
```

Comment:
nit: maybe it would be worth logging / printing some here? Something like `Upserting library model {slug} with commit {commit_sha}`